Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -818,11 +384,11 @@ static HRESULT __DWriteUpdateUserCreatedFontCollection(CFArrayRef datas, CFArray
     RETURN_IF_FAILED(DWriteCreateFactory(DWRITE_FACTORY_TYPE_SHARED, __uuidof(IDWriteFactory), &dwriteFactory));
 
     // Create singleton Font Collection loader, which will be shared for registering/unregistering fonts
-    static ComPtr<DWriteFontCollectionLoader> loader;
-    static HRESULT createdLoader = [](DWriteFontCollectionLoader** collectionLoader) {
+    static ComPtr<DWriteFontBinaryDataCollectionLoader> loader;
```

Comment:
hence the question mark, lol. I personally think its more clear but don't care too much

---
In reply to: [90938094](https://github.com/Microsoft/WinObjC/pull/1504#discussion_r90938094) [](ancestors = 90938094,90935689)